scripts/protect-branches.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+
+
+import argparse
+import json
+import os
+import re
+import requests
+import sys
+import urllib
+
+
+# API token received from github. For the "scopes", I only selected the entire "repo" section.
+# https://github.com/settings/tokens
+TOKEN=""
```

Comment:
Can we adjust this so that it reads from an environment variable. Since this file is under VCS and our edit would add a secret into the file it seems a little dangerous. A little comment saying set the GITHUB_TOKEN env variable would resolve this issue.